<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wToolMenu</class>
 <widget class="QWidget" name="wToolMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>128</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="btnRegion">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Region</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnNullString">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Null terminated string</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnString">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Size limited string</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnErase">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Erase</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnRegion</sender>
   <signal>clicked()</signal>
   <receiver>wToolMenu</receiver>
   <slot>btnRegion_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNullString</sender>
   <signal>clicked()</signal>
   <receiver>wToolMenu</receiver>
   <slot>btnNullString_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnString</sender>
   <signal>clicked()</signal>
   <receiver>wToolMenu</receiver>
   <slot>btnString_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnErase</sender>
   <signal>clicked()</signal>
   <receiver>wToolMenu</receiver>
   <slot>btnErase_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnRegion_clicked()</slot>
  <slot>btnNullString_clicked()</slot>
  <slot>btnString_clicked()</slot>
  <slot>btnErase_clicked()</slot>
 </slots>
</ui>
